Makefile

```diff
@@ -150,7 +150,8 @@ e2e-test-local: kind-up e2e-test
 .PHONY: ginkgo-check
 ginkgo-check:
 ifeq ("$(wildcard $(GINKGO))", "")
-	@echo "ERROR: Need to install ginkgo first, run: go get -u github.com/onsi/ginkgo/v2/ginkgo@v2.9.2 or go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.9.2"
+	@echo "ERROR: Need to install ginkgo first, run: go get -u github.com/onsi/ginkgo/v2/ginkgo@v2.9.2 and go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.9.2"
```

Comment:
The go get installs for versions prior to Go 1.16. 

And it can be installed via go install after 1.16

Reply:
> And it can be installed via go install after 1.16

Thanks. Yes it is. I think the confusion was either or. So I have concatenated it with `and`. `go get` just updates your go mod file. Right?

Reply:
Yes I found it from here https://go.dev/doc/go-get-install-deprecation